packages/winston-log-enricher/README.md

```diff
@@ -34,6 +34,16 @@ format: winston.format.combine(
 )
 ```
 
+**Using log enricher with Node.js agent >= 8.11.0:**
+
+Node.js agent 8.11.0 introduced [application logging](https://docs.newrelic.com/docs/logs/logs-context/configure-logs-context-nodejs/).  By default it will instrument winston automatically and depending on your application logging configuration, it may favor log decorating + in agent forwarding or local log decorating instead of log enrichment.  Log enrichment will still work without any configurations but it is recommended to add the following to your newrelic configuration:
```

Comment:
Maybe something like this?

By default the agent will instrument winston automatically and depending on your application logging configuration, may favor in agent log forwarding with decoration or local log decorating instead of traditional log enrichment.  Log enrichment will still work without any configurations but it is recommended to add the following to your newrelic configuration:

Reply:
This sounds good to me 👍